tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/Program.cs

```diff
@@ -79,6 +79,10 @@ static async Task HandleGenerateCodeFile(IEnumerable<string> swaggers, string ou
             
             
             SwaggerApiViewRoot root = new SwaggerApiViewRoot(packageName, packageName);
```

Comment:
Unrelated to this PR. But why are we passing same variable twice to SwaggerApiViewroot?
`packageName`

Reply:
Cood catch. I will check it.